config/fpga/analog_slice_cfg.yml

```diff
@@ -1,16 +1,16 @@
 dt: 0.1e-6
 func_order: 1
 func_numel: 512
-func_domain: [0.0, 3.19375e-9]
+func_domain: [0.0, 1.0e-9]
```

Comment:
tightened domain for step response to improve accuracy (given a fixed number of PWL segments).  this is still plenty for a time constant of 100ps.